pandas/core/generic.py

```diff
@@ -2742,6 +2744,18 @@ def convert_objects(self, convert_dates=True, convert_numeric=False,
             conversion was done). Note: This is meant for internal use, and
             should not be confused with inplace.
 
+        See also
+        --------
+        pandas.to_datetime
+            Convert argument to datetime.
```

Comment:
Following https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections, the format for the see also is like 

```
pandas.to_datetime : Convert argument to datetime
```